docs/narr/startup.rst

```diff
@@ -130,8 +131,9 @@ Here's a high-level time-ordered overview of what happens when you press
 
 #. ``pserve`` starts the WSGI *server* defined within the ``[server:main]``
    section.  In our case, this is the Waitress server (``use =
-   egg:waitress#main``), and it will listen on all interfaces (``host =
-   127.0.0.1``), on port number 6543 (``port = 6543``). The server code itself
+   egg:waitress#main``), and it will listen on all interfaces (``listen =
+   127.0.0.1:6543 [::1]:6543``, means that it will listen on ipv4 and ipv6),
+   on port number 6543. The server code itself
```

Comment:
Minor grammar fixes. Also you no longer need to hard wrap paragraphs.

and it will listen on all interfaces on port 6543 for both IPv4 and IPv6 (``listen = 127.0.0.1:6543 [::1]:6543``).